glhe/properties/fluid_properties.py

```diff
@@ -39,12 +38,12 @@ def __init__(self, inputs: dict):
         rho_vals = [self.calc_density(x) for x in temps]
         rho_cp_vals = [self.calc_vol_heat_capacity(x) for x in temps]
 
-        self.cp_interp = interp1d(temps, cp_vals)
-        self.k_interp = interp1d(temps, k_vals)
-        self.mu_interp = interp1d(temps, mu_vals)
-        self.pr_interp = interp1d(temps, pr_vals)
-        self.rho_interp = interp1d(temps, rho_vals)
-        self.rho_cp_interp = interp1d(temps, rho_cp_vals)
+        self.cp_interp = Interpolator1D(temps, cp_vals)
+        self.k_interp = Interpolator1D(temps, k_vals)
+        self.mu_interp = Interpolator1D(temps, mu_vals)
+        self.pr_interp = Interpolator1D(temps, pr_vals)
+        self.rho_interp = Interpolator1D(temps, rho_vals)
+        self.rho_cp_interp = Interpolator1D(temps, rho_cp_vals)
```

Comment:
Conceal the interpolation behind an Interpolator class so we only have scipy in one file and it will be relatively easy to modify.